apps/client/src/common/components/input/time-input/TimeInput.tsx

```diff
@@ -26,13 +26,17 @@ function ButtonInitial(name: TimeEntryField) {
   if (name === 'timeStart') return 'S';
   if (name === 'timeEnd') return 'E';
   if (name === 'durationOverride') return 'D';
+  if (name === 'timeWarning') return 'Wa';
+  if (name === 'timeDanger') return 'Da';
```

Comment:
I think we shouldnt use the element with the button for this.
The button was meant to lock the data input (for example to link events), I dont see a use case for actions here